src/tests/Feature/cacheTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Facades\Cache;
+use Lab19\Cart\Testing\TestCase;
+
+class PasswordResetTest extends TestCase
```

Comment:
@AlexeiaW I think this test just needs a rename.

Reply:
done